### ISSUE TITLE
[Question] how droidfs handles deleteing files

### ISSUE THREAD

**Issue by creeper451** [hardcore-sushi/DroidFS#135]
The other day I was importing files into my container when droidfs completed it asked me if I want to wipe the unencrypted files so I was wondering if it actually overwrites the files or it just delete them like any other app without overwriting the actual data on the phone.

**Comment by hardcore-sushi** [comment]
No if you delete with DroidFS the files will be overwritten 2 times before being removed. Maybe you have a better idea of ​​the message that should be displayed to make it clearer?

**Comment by creeper451** [comment]
I think you should add a check box for overwriting the files so in case you need to delete the files quickly and your don't care if there not completely deleted or a simple dialogue (like the disk formatter in windows when you have to select the filesystem) for selecting how many times you want to overwrite the files or not at all
![disk format](https://user-images.githubusercontent.com/92668459/170818050-82f9e161-4322-4bdd-8113-9af7f7b6a924.png)

**Comment by hardcore-sushi** [comment]
If you don't want to overwrite the files, you can simply delete them with a standard file manager. This feature is only there for security.

**Comment by creeper451** [comment]
yes but that would be more convenient for quickly deleting the file without have to go through the file manager.

**Comment by P650SE** [comment]
I agree with OP, it would be nice having a checkbox option in the delete confirmation dialogue to select whether to delete the file quickly or delete with overwrite.
